proteobench/modules/dda_quant/plot.py

```diff
@@ -1,12 +1,14 @@
 import numpy as np
+import pandas as pd
 import plotly.figure_factory as ff
 import plotly.graph_objects as go
 import streamlit as st
 from streamlit_plotly_events import plotly_events
 
 
+# ! This class does not use any instance attributes.
 class PlotDataPoint:
```

Comment:
The PlotDataPoint class has no instance attributes. So two simple function returning the empty figures as currently done in the template should be fine